app/components/shared/inputs/index.ts

```diff
@@ -26,6 +26,7 @@ export interface IInputMetadata {
   hint?: string;
   type?: EInputType;
   title?: string;
+  tooltip?: string;
```

Comment:
it looks like `hint` is the same like `tooltip` can you check that we use the same parameter name for the tooltips before we merge this?

Reply:
`hint` is used nowhere in our repo, for all of the tooltips we've been using `tooltip`

Reply:
if so just delete `hint` from `IInputMetadata` interface